docs/quick-start.md

```diff
@@ -31,6 +31,7 @@ The Jetpack Monorepo requires the following to be installed on your machine:
  	- A public SSH key ([recommended](https://github.com/Automattic/jetpack/blob/trunk/docs/development-environment.md#clone-the-repository)): `git clone git@github.com:Automattic/Jetpack.git` 
 	- HTTPS: `git clone https://github.com/Automattic/jetpack.git` 
 	- If you're not an Automattician, you can [fork the repo following the instructions here](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
+- Note that the Monorepo should not be cloned into the WordPress plugins directory (you will see a warning on your plugins page in that case saying that the Jetpack Monorepo is not a plugin and shouldn't be installed as one). If you are not cloning into a Docker environment, read the [full Development Environment guide here](development-environment.md#clone-the-repository) to find out how to add symlinks.
```

Comment:
Should this be indented to put it under the bullet on line 30? As it is, this reads somewhat oddly. Then again, it already reads somewhat oddly IMO.

> Here's the stuff you need to install:
>
> * Clone our repo.
>    * One method
>    * Another method
>    * Here's a tip if you're not an a11n.
> * Oh wait, here's a big note on where _not_ to clone the repo
> * Another software to install.
> * Another software to install.
> * Weird instruction to change directories. That's not something to install?
> * Another software to install.
> * Another software to install.
> * Another software to install.
> * Another software to install.
> * Another software to install.

Some of that's because this list is mixing "list of stuff to install" with "commands to install it on a Mac".

Reply:
Ah right, yes it should have been indented to be under the bullet on line 30. I can't quite spot how the 'HTTPS' line and the 'If you're not an Automattician' line have an additional indent as they don't appear to viewing the raw file - other than the use of a space for the 'A public SSH key' line which looks invisible. 🤷 

> Some of that's because this list is mixing "list of stuff to install" with "commands to install it on a Mac".

I have a suggestion for this! I'll commit it and comment when it's ready for review.